libwhisper/WhisperHost.h

```diff
@@ -77,8 +73,8 @@ class WhisperHost: public HostCapability<WhisperPeer>, public Interface, public
 	virtual void onStopping() override { stopWorking(); }
 
 	void streamMessage(h256 _m, RLPStream& _s) const;
-
 	void noteChanged(h256 _messageHash, h256 _filter);
+	void advertizeTopicsOfInterest();
```

Comment:
Maybe better to name 'sendTopicsOfInterest()'. This is a network operation and contains no other logic (with respect to advertising).